composer.json

```diff
@@ -28,7 +28,7 @@
         "psr/http-factory-implementation": "^1.0",
         "php-http/discovery": "^1.6",
         "nyholm/psr7": "^1.0",
-        "paragonie/certainty": "^1|^2",
+        "paragonie/certainty": ">= 2.4",
```

Comment:
I'd like to make this follow semver if possible. Thoughts?

```suggestion
        "paragonie/certainty": "^2.4",
```

Reply:
Yeah, that's fine. :)